README.md

```diff
@@ -118,30 +118,30 @@ For more, see the [documentation](https://docs.astral.sh/ruff/).
 
 Ruff is available as [`ruff`](https://pypi.org/project/ruff/) on PyPI:
 
-```shell
-# With uv.
-uv add --dev ruff     # to add ruff to your project
-uv tool install ruff  # to install ruff globally
+```console
+$ # With uv.
+$ uv tool install ruff  # Install Ruff globally.
+$ uv add --dev ruff     # Or add Ruff to your project.
```

Comment:
You can't use leading `$` in the GitHub README without breaking copy/paste.

Reply:
Sad, thanks.